RELEASE_NOTES.md

```diff
@@ -9,7 +9,7 @@ Details about the thing that changed that needs to get included in the Release N
 
 
 # Windows azure-chef-extension 1210.12.109.1004 release notes:
-In this release, we have fixed runlist conflicts with policy_name and policy_group settings in first-boot.json.
+In this release, we have fixed runlist conflicts with policy_name and policy_group settings in first-boot.json, Displaying the runlist when there are no entries, Update readme with microsoft image, Added changes for spliting multiple certificates and Fixing OpenSSL `header too long` issue.

```

Comment:
This can be modified a bit. 
```
In this release, we have fixed some bugs. 

- Fixed runlist conflicts with policy_name and policy_group settings in first-boot.json.
- Added changes for spliting multiple certificates.
- Fixed OpenSSL `header too long` issue.
```